articles/azure-functions/functions-scale.md

```diff
@@ -10,15 +10,15 @@ ms.custom: H1Hack27Feb2017
 ---
 # Azure Functions scale and hosting
 
-When you create a function app in Azure, you must choose a hosting plan for your app. There are three hosting plans available for Azure Functions: [Consumption plan](#consumption-plan), [Premium plan](#premium-plan), and [App Service plan](#app-service-plan).
+When you create a function app in Azure, you must choose a hosting plan for your app. There are three hosting plans available for Azure Functions: [Consumption plan](#consumption-plan), [Premium plan](#premium-plan), and [Dedicated (App Service) plan](#app-service-plan).
 
 The hosting plan you choose dictates the following behaviors:
 
 * How your function app is scaled.
 * The resources available to each function app instance.
-* Support for advanced features, such as VNET connectivity.
+* Support for advanced features, such as Azure Virtual Network (VNet) connectivity.
```

Comment:
I just checked with our internal editorial guidelines, and we aren't supposed to even use VNet (or VNET) 😦 
```suggestion
* Support for advanced features, such as Azure Virtual Network connectivity.
```